GuildedRoseLLCSpec/ItemViewControllerSpec.swift

```diff
@@ -20,67 +20,62 @@ class ItemViewControllerSpec: QuickSpec {
             
             describe("viewDidLoad") {
                 it("creates a data source") {
+                    
                     controller.viewDidLoad()
                     
                     expect(controller.itemCollectionView.dataSource).notTo(beNil())
                 }
                 
                 it("creates a repository") {
+                    
                     controller.viewDidLoad()
                     
                     expect(controller.itemRepository).notTo(beNil())
                 }
                 
                 it("sets the data source to an empty list") {
-                    controller.viewDidLoad()
                     
-                    let dataSource = controller.itemCollectionView.dataSource as? ItemCollectionViewDataSource
+                    controller.viewDidLoad()
                     
-                    expect(dataSource?.items).to(equal([]))
+                    expect((controller.itemCollectionView.dataSource as? ItemCollectionViewDataSource)?.items).to(equal([]))
                 }
             }
             
             describe("viewWillAppear") {
                 it("sets the data source to a static item list") {
-                    
                     let itemRepository = FakeItemRepository()
                     itemRepository.stub(items: Item.testData)
```

Comment:
Right now, you have test data living in the production code, which is a bit strange.

I would refactor this test to not use `Item.testData`. Instead I would just specify the items in the test.